packages/kununu-theme-v3/scss/variables.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Colors
+ */
+
+$black: #000000;
+$butterscotch: #ffbc47;
+$charcoal-grey: #2f3940;
+$dark: #18242b;
+$deep-sky-blue: #1477fd;
+$gross-green: #99c613;
+$ice: #e7e9e9;
+$light-blue-grey: #d1d3d5;
+$light-grey-blue: #b9bdbf;
+$pink-red: #e9004c;
+$slate-grey: #5d656a;
+$white: #ffffff;
+$white-grey: #f8f8f8;
+
+/*
+ * Gradients
+ */
+$green-gradient: linear-gradient(98deg, #b4de37, $gross-green);
+
+/*
+ * Sizes
+ */
+
+$desktop-media: 1025px;
+$tablet-media: 415px;
+$mobile-media: 414px;
```

Comment:
Had to create a mobile-media breakpoint with this size on the project. I know there are more to come (e.g. 320, 375, 412, 414, etc) but for now we only need this one.

Also, look for a better naming? 🤔 

Idk, maybe...
```scss
...
$mobile-media-xs: 320px;
$mobile-media-sm: 375px;
$mobile-media-md: 412px;
$mobile-media-lg: 414px;
...
```

Reply:
I don't know if they are going to be the same for other elements aside the header, maybe in another component it will be:
```
$mobile-media-xs: 330px;
$mobile-media-sm: 376px;
$mobile-media-md: 402px;
$mobile-media-lg: 414px;
```
I would keep here only the ones that are common:
```
$desktop-media: 1025px;
$tablet-media: 415px;
```
And the other ones that specific for each case where they are used.